chromium_src/net/tools/transport_security_state_generator/input_file_parsers.cc

```diff
@@ -39,7 +39,6 @@ constexpr std::string_view kBravePinsJson = R"brave_pins_json({
     // Brave
     { "name": "adblock-data.s3.brave.com", "pins": "brave"},
     { "name": "ai-chat.bsg.brave.com", "pins": "brave"},
-    { "name": "feedback.brave.com", "pins": "brave"},
```

Comment:
I think this line should stay.

Reply:
Let's remove for now since this endpoint terminates at [Cloudflare](https://developers.cloudflare.com/ssl/reference/certificate-pinning/).